### DISCUSSION TITLE
How to run rose stem tests on the new hpc?

### DISCUSSION THREAD

**Discussion by MichaelWhitall** [MetOffice/simulation-systems#229] [UM]
Hi all!

I've been asked (as part of a review of one of my UM tickets) to run the rose stem tests on the ex1a machine, just to check I haven't broken anything on the new hpc.  But if i try to run rose stem --group=ex1a_developer, I get a permission denied error.  Presumably I don't have a login on that machine and need one to run those tests.  How do I go about getting one?

Or alternatively, I now do actually have a login on the EXZ.  Does anyone know how to run UM rose stem on there?

Cheers!
Mike

**Comment by iboutle** [comment]
If you don't have an ex1a login I think it's fine to leave this for the code reviewer?
